src/main/java/com/suse/saltstack/netapi/datatypes/cherrypy/Applications.java

```diff
@@ -0,0 +1,117 @@
+package com.suse.saltstack.netapi.datatypes.cherrypy;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class Applications {
+
+    @SerializedName("Uptime")
+    private double uptime;
+
+    @SerializedName("Bytes Read/Second")
+    private double readsPerSecond;
+
+    @SerializedName("Current Time")
+    private Date currentTime;
+
+    @SerializedName("Server Version")
+    private String serverVersion;
+
+    @SerializedName("Total Time")
+    private double totalTime;
+
+    @SerializedName("Enabled")
+    private boolean enabled;
+
+    @SerializedName("Start Time")
+    private Date startTime;
+
+    @SerializedName("Bytes Written/Second")
+    private double writesPerSecond;
+
+    @SerializedName("Total Bytes Read")
+    private int totalBytesRead;
+
+    @SerializedName("Current Requests")
+    private int currentRequests;
+
+    @SerializedName("Total Requests")
+    private int totalRequests;
+
+    //TODO: implement requests
+    //@SerializedName("Requests")
+    //private Object requests;
```

Comment:
@moio Good point i think its a good idea to utilize `issues` more so far there is only one ( #43 shameless plug) by me which has not seen any discussion yet.
I mostly based that on how the Dates in `Token` got handled which was similarly commented out marked with a todo so i thought it would it would be ok but now i know better :). 

I really like the idea of @isbm adding a `TODO` mark pointing to the issue. Its mostly personal preference tho since i like to use the todolist of IDEs or grepping for todos.

I will remove it completely but i still would like your thoughts on @isbm  suggestion.  :)